### PR TITLE
ZFIN-8972 Create report of ZFIN genes with transcripts that are missing longest transcrtip from ensembl

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: package-lock.json
Comment (rtaylorzfin) [inline package-lock.json:1]: Do we need the package-lock.json in this PR?

**Review by rtaylorzfin — APPROVED** [review]
Looks good

File: package-lock.json
Comment (cmpich) [reply package-lock.json:1]: Not really, but it should go into the source tree, right? It's removing angular.
I am having trouble undoing this file in my commit history.

Comment (rtaylorzfin) [reply package-lock.json:1]: No problem. We can commit it. Maybe we can talk about a better strategy for handling this file later.

Comment (cmpich) [reply package-lock.json:1]: Yes, that file is frequently causing trouble.

**Pull request merged**